### ISSUE TITLE
Fallback mechanism

### ISSUE THREAD

**Issue by kshorenicholas** [logesixas96/Android-QR-System#4]
Should there be a fallback method to which users can sign in to the event in the event they are unable to have access to the application?

**Comment by logesixas96** [comment]
Do you mean an alternative way to take attendance of the user suppose the user do not have access to the app?

**Comment by kshorenicholas** [comment]
Indeed. We often do the same thing in university where if we do not have access to the app, we can use the browser to sign in instead.

**Comment by logesixas96** [comment]
Will figure it out and let you know.

**Comment by logesixas96** [comment]
The implementation is quite complicated. Still trying to figure it out. Do let me know if you have any suggestion.

**Comment by kshorenicholas** [comment]
True. It can get complicated fast. We could implement android webview but that would require many changes into the codebase.

**Comment by kshorenicholas** [comment]
To be honest, this feature is not within the scope of the project and can be implemented later down the line. But it would be a "nice to have" feature.
